### PR TITLE
Pruzhinina second homework

### PR DESCRIPTION
**Pull request by arinapruzhinina**

### CODE REVIEW

File: hw_http/README.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+##### 1. To do POST you should:
+* paste the url http://127.0.0.1:8001/cities
+* in fields "Headers" you should paste:
+in field key paste: "Authorization",
+in field value paste: "admin {537dc091-cbeb-4802-84f8-46f0ea57694c}"
+* in fields "Body" you must choose a raw(JSON) and paste json like that:
+```
+{
+    "city": "some city", 
+    "country": "some country",
```
Comment (siriusdevs) [inline hw_http/README.md]: this is not a valid json - there can be no comma at the end of the last line

File: hw_http/README.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+* in fields "Body" you must choose a raw(JSON) and paste json like that:
+```
+{
+    "city": "some city", 
+    "country": "some country", 
+    "language": "language",
+    "population": 1000   
+}
+```
+In this filds you should write changes, that you want to do.
```
Comment (siriusdevs) [inline hw_http/README.md]: filds -> field.
everywhere in this readme :)

File: hw_http/README.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+## And click the link: http://127.0.0.1:8001
+
+---
+#### After this command you fill see the main page. You can choose which page to go to. 
+---
+### If you choose the page with name "cities", you will see different cities and some infromation about them, like country, local language and population. 
+
+#### You can open Postman and try to do POST/PUT/DELETE requests. Folows this steps:
+
+##### 1. To do POST you should:
```
Comment (siriusdevs) [inline hw_http/README.md:59]: your post request finds first record with this data and returns its id. but it should return the id of the created content.
you should use `insert... returning id` instead. see our classwork.

**Review by siriusdevs — CHANGES_REQUESTED** [review]

File: hw_http/README.md (hunk 3 above)
Comment (arinapruzhinina) [reply hw_http/README.md:59]: fixed

File: hw_http/README.md (hunk 2 above)
Comment (arinapruzhinina) [reply hw_http/README.md]: fixed

File: hw_http/README.md (hunk 1 above)
Comment (arinapruzhinina) [reply hw_http/README.md]: fixed

**Pull request merged**